lightning/src/util/events.rs

```diff
@@ -135,6 +136,14 @@ pub enum Event {
 		/// The outputs which you should store as spendable by you.
 		outputs: Vec<SpendableOutputDescriptor>,
 	},
+	/// Used to indicate that a channel was closed at the given timestamp.
```

Comment:
What timestamp?

Reply:
Errata of mine, previously thought incorporating timestamp could be useful. Like height when the event has been generated. If something is wrong and you don't see a closing after X, raise a bell ? 

I think such feature could be introduced latter on, if users require it.